tests/perfalert/test_auto_perf_sheriffing/test_perf_sheriff_bot.py

```diff
@@ -153,26 +61,32 @@ def test_assert_can_run_doesnt_throw_exception_when_enough_time_left(
 
 
 def test_records_and_db_limits_remain_unchanged_if_no_records_suitable_for_backfill(
-    report_maintainer_mock, backfill_tool_mock, secretary, sheriff_settings, preliminary_record
+    report_maintainer_mock,
+    backfill_tool_mock,
+    secretary,
+    notify_client_mock,
+    sheriff_settings,
+    record_unsuited_for_backfill,
 ):
     sheriff_bot = PerfSheriffBot(
-        report_maintainer_mock, backfill_tool_mock, secretary, **FEATURE_FLAGS
+        report_maintainer_mock, backfill_tool_mock, secretary, notify_client_mock
```

Comment:
We should also assert if notify_client_mock.email gets called/does not get called where necessary.